docs/CHANGELOG.md

```diff
@@ -1,5 +1,42 @@
 # Changelog
 
+
+## v0.20.0
+
+*__NOTE:__ This will be the final release of python-tuf that includes the
+legacy implementation code. Please see the [*1.0.0
+announcement*](1.0.0-ANNOUNCEMENT.md) page for more details about the next
```

Comment:
Not sure if that was a mistake but the 1.0.0 announcement ended up in root dir so this link does not work. Do we move the announcement?

Reply:
🤦  Thanks for catching this!